serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -1,5 +1,13 @@
 = {page-component-title}
 
+{product_name} is a tool to build cloud-native workflow applications. {product_name} can be used to do services and events orchestration and choreografy. Currently, with {product_name} you can integrate with services and events in your architecture using:
```

Comment:
what choreografy means in this context?

Reply:
Here: https://en.wikipedia.org/wiki/Service_choreography

Reply:
I see, thanks.